<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>General</class>
 <widget class="QWidget" name="General">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <property name="accessibleName">
   <string>General</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="groupingAndSportingLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Grouping &amp; Sorting</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="groupingLabel">
     <property name="text">
      <string>&amp;Grouping:</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
     <property name="buddy">
      <cstring>groupingStrategy</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="groupingStrategy">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="onlyGroupWhenFull">
       <property name="text">
        <string>Only when the taskbar is &amp;full</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="sortingLabel">
     <property name="text">
      <string>&amp;Sorting:</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
     <property name="buddy">
      <cstring>sortingStrategy</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="sortingStrategy"/>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="filtersLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Filters</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="showOnlyMinimized">
       <property name="text">
        <string>Only show tasks that are &amp;minimized</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="showOnlyCurrentDesktop">
       <property name="text">
        <string>Only show tasks from the current &amp;desktop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="showOnlyCurrentScreen">
       <property name="text">
        <string>Only show tasks from the current s&amp;creen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="middleClickActionLabel">
     <property name="text">
      <string>Middle Click &amp;Action:</string>
     </property>
     <property name="buddy">
      <cstring>middleClickAction</cstring>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QComboBox" name="middleClickAction">
     <property name="toolTip">
      <string>Action that should be performed when a task item is middle clicked.</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="showOnlyCurrentActivity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>O&amp;nly show tasks from the current activity</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>groupingStrategy</tabstop>
  <tabstop>onlyGroupWhenFull</tabstop>
  <tabstop>sortingStrategy</tabstop>
  <tabstop>showOnlyMinimized</tabstop>
  <tabstop>showOnlyCurrentDesktop</tabstop>
  <tabstop>showOnlyCurrentScreen</tabstop>
  <tabstop>middleClickAction</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
